src/helpers/colors.ts

```diff
@@ -9,12 +9,13 @@ function djb2Hash(str: string): number {
   return hash >>> 0 // ensure non-negative integer
 }
 
-export function getColorFromName(name: string) {
+export function getColorFromName(name: string, darkenColors?: boolean) {
```

Comment:
Now that I see it, it might be better to use a transformer instead of hardcoding this in the helpers. Using this instead will keep everything in their place, and allow for better customization.

```suggestion
export function getColorFromName(name: string, transform?: (color: TinyColor) => TinyColor) {
```